### ISSUE TITLE
Indicate that a Release is active when viewing the Features

### ISSUE THREAD

**Issue by pzaich** [codeamp/panel#171]
When viewing the Features panel, it's easy to forget that a current feature is already being deployed in a pending release. Add some kind of indication that a feature is part of a pending release or add an indicator to sidebar that a release is pending.

**Comment by capripot** [comment]
Yes, many times I too double release a new commit from the features tab because it doesn't tell if it's already being released by Continuous Deploy or by somebody else.

**Describe the solution you'd like**
Two things:
- being able "unqueue" a release
- seeing a flag/tag on the feature page that links to release page and that states "feature being released"
